src/physics.F90

```diff
@@ -390,6 +399,20 @@ subroutine scatter(p, i_nuclide)
     ! Set event component
     p % event = EVENT_SCATTER
 
+    ! sample new outgoing angle for isotropic in lab scattering
+    if (materials(p % material) % p0(i_nuc_mat)) then
+
+      ! sample isotropic-in-lab outgoing direction
+      uvw_new(1) = TWO * prn() - ONE
+      phi = TWO * PI * prn()
+      uvw_new(2) = cos(phi) * sqrt(ONE - uvw_new(1)*uvw_new(1))
+      uvw_new(3) = sin(phi) * sqrt(ONE - uvw_new(1)*uvw_new(1))
+      mu_lab = dot_product(uvw_old, uvw_new)
+
+      ! change direction of particle
+      p % coord(1) % uvw = rotate_angle(uvw_new, mu_lab)
```

Comment:
`uvw_new` already holds the correct values for the particle's `uvw`.  The rotate function would resample `phi` inconsistently with the calculated `mu_lab` and should take `uvw_old` as an argument.  The latter is not really germane; `p % coord(1) % uvw` can just be set to `uvw_new`.

Also, `p % mu` should be set to `mu_lab` here.  Otherwise, it still holds the 'true physics' value computed by whichever scattering subroutine was called above.